docs/appguide.rst

```diff
@@ -95,32 +97,40 @@ A backgrounded application enters the ``ACTIVE`` state via a call to
 :py:meth:`~.TemplateApp.foreground`. When it is active the application owns the
 screen and must draw and maintain its user interface.
 
-If the system manager wants to put the watch to sleep then it will tell the
-active application to :py:meth:`~.TemplateApp.sleep`.
-If the application returns True then the application will remain active
-whilst the watch is asleep.
-It will receive no events nor the application tick whilst the system is
-asleep and, instead, must wait for a :py:meth:`~.TemplateApp.wake` notification
-telling the application that the device is waking up and that it may
-update the screen if needed.
-
-If an application does not support sleeping then it can simply not implement
-:py:meth:`~.TemplateApp.sleep` or :py:meth:`~.TemplateApp.wake`.
-In this case the system manager will automatically return to the default
-application, typically the main clock face.
-
-Some applications may support sleeping only under certain circumstances. For
-example a stopwatch may choose to remain active when the watch sleeps only if
-the stopwatch is running.
-This type of application must implement :py:meth:`~.TemplateApp.sleep` and
-return False when it does not want to remain active when the system
-resumes.
+If the screen timeout is near, the system manager will want to turn the screen
+off to save power. It will call the sleep entry point of the currently 
```

Comment:
I don't think it is a clarification to replace "put the watch to sleep" with "turn the screen off to save power". During a sleep we currently turn off backlight, display controller, touch panel and CPU. In the future we may also want to turn off the BLE advertisements too if there is not a currently active bluetooth connection.

Reply:
Ah shoot, I was convinced that sleeping only changes wether the app will return to the home menu or not.

Reply:
The phrasing can be improved but it should focus on user experience.

~~~
If the user does not interact with the device for a period of time then the system manager will put the device to sleep and turn off the screen to save power. It will call the :py:meth:`~.TemplateApp.sleep` entry point of the currently active application to notify the app that the device will shortly go to sleep.